parser_library/src/expressions/conditional_assembly/terms/ca_expr_list.cpp

```diff
@@ -73,11 +73,7 @@ void ca_expr_list::apply(ca_expr_visitor& visitor) const { visitor.visit(*this);
 
 context::SET_t ca_expr_list::evaluate(const evaluation_context& eval_ctx) const
 {
-    assert(expr_list.size() <= 1);
-
-    if (expr_list.empty())
-        return context::SET_t(expr_kind);
-    return expr_list.front()->evaluate(eval_ctx);
+    return expr_list.size() == 1 ? expr_list.front()->evaluate(eval_ctx) : context::SET_t(expr_kind);
```

Comment:
Was the assert triggered?

Reply:
It was triggered in debug mode for something like this:
```
         AIF   ( (1 AND 1).AAAA).NAMERR
```

Reply:
That would indicate that the resolve_expression/resolve_expression_tree was not called somewhere...